docs/pages/reference/monitoring/audit.mdx

```diff
@@ -159,7 +159,7 @@ For desktop sessions the recording includes the contents of the screen.
 <Tabs>
 <TabItem scope={["enterprise", "oss"]} label="Self-Hosted">
 
-Teleport can store the recorded sessions in an [AWS S3 bucket](../backends.mdx)
+Teleport can store the recorded sessions in a [AWS S3 bucket, Google Cloud storage, Azure Blob Storage](../backends.mdx), 
```

Comment:
I think this needs more work:

- the link text is getting long
- the next paragraph describes the local backend, but this section is not specific to any particular backend